components/charts/ChartBase/index.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from "prop-types";
+
+const ChartBase = ({ children, className, width = 900, height = 300 }) => {
+  return (
+    <svg
+      preserveAspectRatio="xMidYMid meet"
+      viewBox={`0 0 ${width} ${height}`}
+      className={className}
+      style={{ minHeight: height, height: "100%" }}
```

Comment:
this combo of height and minheight overflow the chart if it's wider than it's parent, the viewBox maintains aspect ratio